### DISCUSSION TITLE
New community idea

### DISCUSSION THREAD

**Discussion by mackreal** [city41/smaghetti#118] [Ideas]
I think I should suggest one more thing to form a community and that is, should we propose a Discord server? I assume you have not heard of it but we can discuss about smaghetti in real time.

**Comment by city41** [comment]
This is a pretty common request. The thing is I really don't like real time chat. I know that frustrates everyone else, but it's true :) So if there was a discord server, I would never be on it.

**Reply by mackreal** [reply]
So you're more like a traditional guy? I see.

While a discord is nice to have, I respect your decision but makes me wonder why oppose to that idea.

**Reply by city41** [reply]
To be honest I just don't think real time chat is very effective and can be a huge time sink. Especially for me as the starter of this project. I'd likely get pinged with questions that are more easily answered here, and the pinging would increase as smaghetti gets more known. I also don't like how real time chat is so ephemeral, good content just disappears off the top of the screen and is hard to find with search. Or worse, someone asks a question while the person who knows the answer isn't around, so their question just scrolls off the top and is lost. At least here with discussions and issues, content is not easily lost.

If smaghetti gets more popular, and especially if more people come onto the project, chat might work. But currently smaghetti is quite small. Not many people have signed up, only 21 people in the subreddit, etc. It's very slowly spreading, which I think is fine. I don't think it needs to be this massively popular thing.

**Reply by city41** [reply]
MrSkittles started a smaghetti discord server: https://discord.gg/wBVE4yyWhM
